homeassistant/components/dnsip/sensor.py

```diff
@@ -48,23 +53,46 @@ async def async_setup_platform(
     discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
     """Set up the DNS IP sensor."""
-    hostname = config[CONF_HOSTNAME]
-    name = config.get(CONF_NAME)
-    ipv6 = config[CONF_IPV6]
+    _LOGGER.warning(
+        "Loading dnsip via platform setup is deprecated; Please remove it from your configuration"
```

Comment:
Let's make this a little bit more self-explaining / user-friendly.

```suggestion
        "Configuration of the DNS IP platform in YAML is deprecated and will be "
        "removed in Home Assistant 2022.4; Your existing configuration "
        "has been imported into the UI automatically and can be safely removed "
        "from your configuration.yaml file"
```